ocs_ci/deployment/deployment.py

```diff
@@ -217,6 +217,52 @@ def do_deploy_submariner(self):
             submariner = Submariner()
             submariner.deploy()
 
+    def deploy_gitops_operator(self, switch_ctx=None):
```

Comment:
As discussed earlier, please make sure to cover appset based apps also.

Reply:
added